### ISSUE TITLE
Support for backend

### ISSUE THREAD

**Issue by AndresXI** [AndresXI/Netflix-Clone#10]
I am thinking of making this project a full stack web app. We could build a backend with NodeJS and MongoDB (suggestions are welcome). If you are interested chime in some ideas!

**Comment by JamesWBS** [comment]
yes would love to! want to make it better than netflix or only same
i have a few real domains we could use later if we can get it working like
real site, as a showcase
and real content from filmmaker friends
can we add a transaction vod and subscription vod etc_
i good one to replicate if u really want to go crazy is muvi.com

**Comment by AndresXI** [comment]
Yeah let's do it. We can always change directions if we need to. We can start with some fake data and start from there.

**Comment by alexandreliberato** [comment]
I would suggest to keep it as a frontend project and create just a API doc for other people create the backend using their favorite stack like golang, python, etc.

**Comment by michaelchora** [comment]
@AndresXI I could build a Laravel backend for this pretty easily. I hit you via email. Keen to have a go!

**Comment by lollykrown** [comment]
This is beautiful I must say. kudos! I can create a backend or rather I'm will to collaborate on a backend with NodeJs and MongoDb. But the question is where would the real movies come from?

**Comment by AndresXI** [comment]
The backend would be for the sole purpose to have users. And these users can review, comment, create playlists, and rate movies. We would keep using The Movie Database API though, we wouldn't actually host any videos.

**Comment by lollykrown** [comment]
I will start working on it hopefully by next week.

**Comment by lucifer-kj** [comment]
I wanted to know whether it's possible to host videos in this app. If yes, then can someone please help me with that.

**Comment by devandres-tech** [comment]
This frontend app does not necessarily hosts videos - although this could be a great idea for a new feature...

**Comment by Poseidon-G** [comment]
Hope have repo for this issue, I will join to contribute for u

**Comment by ChakraZulu77** [comment]
Maybe should be a movie rating site with only legal free to share previews and clips from shows and movies to be commented and debated upon. Kind of an IMDb but more interactive socially. In the future possibly feature up and coming filmaker's work for prescreening events via the www?

**Comment by ChakraZulu77** [comment]
Within the site can have clubs or group pages such as the "Marvel Universe Group Page" ,"Fans of Quentin Tarantino", etc. 
You're Welcome....💯😉🤫
